nmap-service-probes

```diff
@@ -15473,3 +15473,22 @@ Probe TCP VersionRequest q|VERSION|
 ports 3310
 rarity 8
 match clam m|^ClamAV ([\w.]+)/(\w+)/(.+)$| p/ClamAV/ v/$1 ($2)/ i/AV definitions updated on:$3/
+
```

Comment:
Ok, it looks like RSE gets the same or similar response to the LDAPBindReq probe as well as an amusing `server failure: Authentification Failed` response to 3 other probes (ajp, redis-server, and hazelcast-http). Is there a reason not to mark one of those with port 4035 and use it instead? Did the service experience any hangs or crashes due to the unmodified scan? (The JMON service did not respond to any other probes, so this is really only about RSE)